### DISCUSSION TITLE
Installing chocolatey package to 2 destinations

### DISCUSSION THREAD

**Discussion by owenfunkhouser** [chocolatey/choco#2700] [Q&A] [answered]
I have a chocolatey package with an installChocolatey.ps1 file that calls Get-ChocolateyUnzip twice with two different destinations. When uninstalling the package, only one of the destinations is removed. Any tips on how to fix this? I am trying to avoid having two chocolatey packages, one for each destination folder.

**Comment by TheCakeIsNaOH** [comment, marked as answer]
Does this package have an uninstall script, and if so, what does it contain?

You would need either a pair of `Uninstall-ChocolateyZipPackage` calls to remove the two folders, or two `Remove-Item` calls. If it is the `Uninstall-ChocolateyZipPackage`, then you would need to use different a `-PackageName` for each call to it (and corresponding changes to  `Get-ChocolateyUnzip`).

**Reply by owenfunkhouser** [reply]
The different package name is the solution. Thank you!
